### PR TITLE
Front page

### PR DESCRIPTION
**Pull request by heqiu**
1.  added buttons to navigate to other views
2.  removed pills

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  layout "home"
```
Comment (brentwalther) [inline app/controllers/home_controller.rb]: You shouldn't need to specify this.

File: app/views/home/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:css
+  #title  {padding:5% 0; font-size:40px;}
+  #buttons {padding: 6% 0;}
+#title
```
Comment (brentwalther) [inline app/views/home/index.html.haml:4]: Instead of adjusting font size manually, just use an `h1` instead of a `p`

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'home/index'
```
Comment (brentwalther) [inline config/routes.rb]: What is this for?

File: features/step_definitions/home_navigation.feature [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+  Scenario: User wants to view evaluation data
+    Given User is on the home page
+    When User clicks on the evaluation data button
+    Then User should see the evaluations page
+
+  Scenario: User wants to view faculty member historical data
+    Given User is on the home page
+    When User clicks on the faculty member historical data button
+    Then User should see the faculty member historical data page
```
Comment (brentwalther) [inline features/step_definitions/home_navigation.feature]: Please add a newline at the end of the file.

File: features/step_definitions/home_navigation.feature [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Feature: Be able to navigate to other pages from home page
+
+  Scenario: User wants to view evaluation data
```
Comment (brentwalther) [inline features/step_definitions/home_navigation.feature:9]: Should add one additional scenario that tests what content is on the home page. Feel free to do either here or in Rspec

File: spec/views/home/index.html.haml_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (brentwalther) [inline spec/views/home/index.html.haml_spec.rb]: You can just delete this file.

File: config/routes.rb (hunk 3 above)
Comment (heqiu) [inline config/routes.rb]: Not sure, don't actually remember adding it.  I removed it.

File: features/step_definitions/home_navigation.feature (hunk 4 above)
Comment (heqiu) [inline features/step_definitions/home_navigation.feature]: Will do.  Is it generally good practice to always have an empty line at the end?

Comment (brentwalther) [inline features/step_definitions/home_navigation.feature]: Yep.

**Pull request merged**